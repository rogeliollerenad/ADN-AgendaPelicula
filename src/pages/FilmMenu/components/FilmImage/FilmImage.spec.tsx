import React from 'react';
import { render } from '@testing-library/react';
import { FilmImage } from '.';

describe('FilmImage Test', () => {
  it('should match snapshot', () => {
    const { container } = render(<FilmImage cover="film.jpg" />);
    expect(container).toMatchSnapshot();
  });
});
