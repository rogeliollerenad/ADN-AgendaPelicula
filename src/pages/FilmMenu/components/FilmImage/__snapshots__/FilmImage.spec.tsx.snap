// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmImage Test should match snapshot 1`] = `
<div>
  <img
    alt="Film item"
    src="/images/film.jpg"
  />
</div>
`;
